src/components/task-plan/index.cjsx

```diff
@@ -82,7 +71,6 @@ PlanShell = React.createClass
       id={id}
       store={TaskPlanStore}
       actions={TaskPlanActions}
-      saved={@saved}
```

Comment:
hmm, why not always `transitionTo` the calendar from here?

And if not possible, then is there any other code that uses `<LoadableItem>` `saved` prop?